src/store/app/app.actions.js

```diff
@@ -1,9 +1,22 @@
+import { isNil } from 'lodash'
+
 export default {
   /**
    * Closes "add to home screen" modal for apple
    */
   closeAddToHomeScreenModalForApple: async ({ commit }) => {
     localStorage.setItem('addToHomeIosPromptLastDate', Date.now())
     commit('setShowAddToHomeScreenModalForApple', false)
+  },
+
+  /**
+   * Trigger service worker skipWating so the new service worker can take over.
+   * This will also trigger a window refresh (see /src/misc/register-service-worker.js)
+   */
+  serviceWorkerSkipWaiting({ state, commit }) {
+    if (isNil(state.SWRegistrationForNewContent)) return
+
+    commit('setRefreshingApp', true)
+    state.SWRegistrationForNewContent.waiting.postMessage('skipWaiting')
```

Comment:
Is this the part that send 'skipWaiting' to the service worker ? Where can I find a registration structure, I can't find it. The `waiting` part seems to get ou from space.

Reply:
> Is this the part that send 'skipWaiting' to the service worker ?
Yes 

> Where can I find a registration structure, I can't find it. The waiting part seems to get ou from space.
`register-service-worker.js`